scripts/run-sync.js

```diff
@@ -0,0 +1,18 @@
+const path = require('path');
+const uploadAPISpecification = require('./upload-api-specification');
+const core = require('@actions/core');
+const github = require('@actions/github');
+
+(async () => {
+  //fetch the changed files
```

Comment:
There's a finite number of API specs we'll ever upload, it might be better to pull out all this changed files logic and sync them all?

Reply:
We implemented this logic to make sure that the open api files that does not have a change will not get overwritten again by the GH action.. So, is it okay if we just blindly update all the files and overwrite them when there is actually not any changes to the readme doc file?

Reply:
sure Colm.. Missed that scenario while implementing this 😅 will implement that in the follow up PR.